Resources/public/js/views/dashboard/ez-dashboardblockallcontentview.js

```diff
@@ -158,20 +86,16 @@ YUI.add('ez-dashboardblockallcontentview', function (Y) {
                     limit: 10
                 }
             });
-        },
-
-        destructor: function () {
-            this._clearClickOutsideHandler();
         }
     }, {
         ATTRS: {
             /**
-             * The items list.
+             * The block content - items list.
              *
-             * @attribute items
+             * @attribute content
              * @type Array of objects containing location structs
              */
-            items: {
```

Comment:
actually the dashboard asynchronous view now provide the attribute, so this can be completely removed